### PR TITLE
:pencil2: Add logging

### PR DESCRIPTION
**Pull request by PaulMathon**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ const healthCheck = require('express-healthcheck');
 const { runHealthChecks } = require('./health-checks');
 const swaggerUi = require('swagger-ui-express');
 const { NewmanRunner } = require('./runner');
+const morgan = require('morgan');
```
Comment (ouvreboite) [inline app.js:14]: why morgan?

File: app.js [hunk 2]
```diff
@@ -66,6 +84,11 @@ class Application {
           ? JSON.parse(req.files.iterationDataFile.data.toString())
           : null;
 
+        logger.log(
+          LogLevel.info,
+          `Running Postman collection: ${collectionFileJSON.info.name}`
```
Comment (ouvreboite) [inline app.js:89]: can you "null protect" this, I don't want the server to crash is the collectionFile does not have any "info" prop

File: logger.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+const winston = require('winston');
```
Comment (ouvreboite) [inline logger.js:1]: why winston? Do we need both winston and morgan?

File: constants.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const APPLICATION_NAME = process.env['APPLICATION_NAME'] || '';
+const INSTANCE_ID = process.env['INSTANCE_ID'] || '';
+const INSTANCE_NAME = process.env['INSTANCE_NAME'] || '';
```
Comment (ouvreboite) [inline constants.js:3]: This means we rely (or offer the feature) on env variable for more complete logs.
So we should add a small section in the readme explaining what the logs looks like and that we can inject APPLICATION_NAME, INSTANCE_ID and INSTANCE_NAME in them.

**Review by ouvreboite — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (PaulMathon) [reply app.js]: Done

File: constants.js (hunk 4 above)
Comment (PaulMathon) [reply constants.js:3]: Done

**Review by ouvreboite — APPROVED** [review]

**Pull request merged**